ballerina/tests/simple-params-query-test.bal

```diff
@@ -581,6 +581,26 @@ function queryTimeStringInvalidParam() {
     }
 }
 
+type InvalidRecord record {
+    int id;
+};
+
+@test:Config {
+    groups: ["query", "query-simple-params"]
+}
+function queryInvalidRecordParam() {
+    InvalidRecord recordValue = {id : 1};
+    ParameterizedQuery sqlQuery = `SELECT * from DataTable WHERE row_id = ${recordValue}`;
+    record {}|error? result = trap queryMockClient(simpleParamsDb, sqlQuery);
+    test:assertTrue(result is error);
+    io:println(result);
```

Comment:
Is the print statement necessary?

Reply:
corrected in https://github.com/ballerina-platform/module-ballerina-sql/pull/358/commits/2e7b8240f65198e3eb8a641457944d7b0bdfe7e9